beacon-chain/blockchain/service.go

```diff
@@ -195,6 +199,9 @@ func (c *ChainService) Stop() error {
 // Status always returns nil.
 // TODO(1202): Add service health checks.
 func (c *ChainService) Status() error {
```

Comment:
who call status and what are we doing with its response

Reply:
https://github.com/prysmaticlabs/prysm/blob/master/shared/prometheus/service.go#L47

Reply:
lg